lib/netuitive/rails_config_manager.rb

```diff
@@ -1,29 +1,73 @@
 require 'netuitive/netuitive_rails_logger'
 class ConfigManager
 
-	class << self
-		def setup()
-			readConfig()
-		end
-
-		def readConfig()
-			gem_root= File.expand_path("../../..", __FILE__)
-			data=YAML.load_file "#{gem_root}/config/agent.yml"
-			debugLevelString=ENV["NETUITIVE_RAILS_DEBUG_LEVEL"]
-			if(debugLevelString == nil or debugLevelString == "")
-				debugLevelString=data["debugLevel"]
-			end
-			if debugLevelString == "error"
-				NetuitiveLogger.log.level = Logger::ERROR
-			elsif debugLevelString == "info"
-				NetuitiveLogger.log.level = Logger::INFO
-			elsif debugLevelString == "debug"
-				NetuitiveLogger.log.level = Logger::DEBUG
-			else
-				NetuitiveLogger.log.level = Logger::ERROR
-			end
-			NetuitiveLogger.log.debug "read config file. Results: 
-				debugLevel: #{debugLevelString}"
-		end
-	end
-end 
+  class << self
+    def setup()
+      readConfig()
+    end
+
+    def captureErrors
+      @@captureErrors
+    end
+
+    def ignoredErrors
+      @@ignoredErrors
+    end
+
+    def queueTimeDivisor
+      @@queueTimeDivisor
+    end
+
+    def readConfig()
+      gem_root= File.expand_path("../../..", __FILE__)
+      data=YAML.load_file "#{gem_root}/config/agent.yml"
+      debugLevelString=ENV["NETUITIVE_RAILS_DEBUG_LEVEL"]
+      if(debugLevelString == nil or debugLevelString == "")
+        debugLevelString=data["debugLevel"]
+      end
```

Comment:
Not sure if I get the intention here, but from what I see you could just do
```
debugLevelString = ENV['NETUITIVE_RAILS_DEBUG_LEVEL'].presence || data['debugLevel']
```
This would assign the value of the environment variable to `debugLevelString` only if it is present (which means not nil and not blank) otherwise it would assign the content of the YAML entry